roles/horizon/tasks/main.yml

```diff
@@ -173,6 +165,60 @@
     - horizon.favicon_url is defined
     - openstack_install_method == 'distro'
 
+- name: Install horizon-customization module (rhel osp)
```

Comment:
Should we wrap below task under a block with when condition for os =='rhel'

Do we need this customization to be laid down for both centos(source) and rhel(distro) i.e os == 'rhel' ?